crypto/src/transaction_context/mod.rs

```diff
@@ -215,19 +214,11 @@ impl TransactionContext {
         result
     }
 
-    /// Initializes the MLS client if [super::CoreCrypto] has previously been initialized with
-    /// `CoreCrypto::deferred_init` instead of `CoreCrypto::new`.
-    /// This should stay as long as proteus is supported. Then it should be removed.
-    pub async fn mls_init(
-        &self,
-        identifier: ClientIdentifier,
-        ciphersuites: Vec<MlsCiphersuite>,
-        nb_init_key_packages: Option<usize>,
-    ) -> Result<()> {
-        let nb_key_package = nb_init_key_packages.unwrap_or(INITIAL_KEYING_MATERIAL_COUNT);
+    /// Initializes the MLS client of [super::CoreCrypto].
+    pub async fn mls_init(&self, identifier: ClientIdentifier, ciphersuites: Vec<MlsCiphersuite>) -> Result<()> {
```

Comment:
There's no particular reason for `ciphersuites` to need an owned vector, that I can tell. Why not `&[MlsCiphersuite]`?